### ISSUE TITLE
No blurred (black, opacity 0.5) background

### ISSUE THREAD

**Issue by wollodev** [compact/angular-bootstrap-lightbox#13]
Modal-backdrop isn't correctly displayed. I think the height of 0px is to small and that's the reason why the whole background layer is missing.

Please compare bootstrap modal with your demo for proof.

Thank you.

**Comment by magiccrafter** [comment]
This is a problem introduced in Bootstrap version 3.3.1.
This is the temporary fix:
```javascript
.modal-backdrop {
    bottom: 0;
}
```
Check this for details: angular-ui/bootstrap/issues/2970

**Comment by mattuuh7** [comment]
@magiccrafter this works. For longer pages, i need to put it to 
```
.modal-backdrop {
    bottom: -1000;
}
```

But it works. Thanks!

**Comment by compact** [comment]
This is fixed in Bootstrap 3.3.4. No need for the extra `.modal-backdrop` rule now.

**Comment by mattuuh7** [comment]
thanks
